aibolit/patterns/classic_setter/classic_setter.py

```diff
@@ -11,24 +8,38 @@ class ClassicSetter:
     def __init__(self):
         pass
 
-    def value(self, filename: str) -> List[LineNumber]:
-        lst: List[LineNumber] = []
-        tree = build_ast(filename).filter(javalang.tree.MethodDeclaration)
-        for _, node in tree:
-            if (node.return_type is None) and ('set' in node.name[:3]):
-                if (isinstance(node.body, list)) and len(node.body) < 2:
-                    for statement in node.body:
-                        if isinstance(statement, javalang.tree.StatementExpression):
-                            if isinstance(statement.expression, javalang.tree.Assignment):
-                                expression = statement.expression.expressionl
-                                if isinstance(expression, javalang.tree.This):
-                                    if statement.expression.type == '=':
-                                        if expression.selectors[0].member.lower() == node.name.lower()[3:]:
-                                            lst.append(node._position.line)
-                                    else:
-                                        break
-                                else:
-                                    break
-                            else:
-                                break
-        return lst
+    def check_nodes(self, ast: AST, body_nodes: List) -> bool:
```

Comment:
We better make here more meaningful name like `check_setter_body`.

Reply:
Also `List` type should have a parameter.

Reply:
It might even accept a single node of `METHOD_DECLARATION` and access its `bode` field inside this method.

Reply:
ok